discovery-provider/plugins/pedalboard/apps/relay/src/index.ts

```diff
@@ -1,22 +1,21 @@
 import App from "basekit/src/app";
 import { webServer } from "./server";
 import { Config, readConfig } from "./config";
-import { ethers } from "ethers";
-import { WalletManager, regenerateWallets } from "./walletManager";
+import { Wallet, ethers } from "ethers";
 import { logger } from "./logger";
 import { initializeDiscoveryDb } from "basekit/src";
 import { connectWeb3 } from "./web3";
 
 export type SharedData = {
   config: Config;
   web3: ethers.providers.JsonRpcProvider;
-  wallets: WalletManager;
+  operatorWallet: Wallet;
```

Comment:
we refer to this as delegate owner waller elsewhere, maybe we should adopt that terminology too? (i don't like it but for consistency sake)